Client/src/Unified/Services/Submitter/Service.cs

```diff
@@ -229,6 +229,23 @@ public class Service : AbstractClientService, ISubmitterService
   public string SessionId
     => SessionService.SessionId.Id;
 
+  /// <inheritdoc />
+  [PublicAPI]
+  public string Submit(string                    methodName,
+                       object[]                  arguments,
```

Comment:
This was voluntarily removed and replaced by the method extensions (hence the includes in all the other files).